src/pages/crag/[id].tsx

```diff
@@ -84,101 +84,9 @@ export const getStaticProps: GetStaticProps<CragProps, { id: string }> = async (
       notFound: true
     }
   }
-  const query = gql`query AreaByID($uuid: ID) {
-    area(uuid: $uuid) {
-      id
-      uuid
-      areaName
-      gradeContext
-      media {
-        mediaUrl
-        mediaUuid
-        destination
-        destType
-      }
-      totalClimbs
-      aggregate {
-        byGrade {
-          count
-          label
-        }
-        byDiscipline {
-            sport {
-              total
-            }
-            trad {
-              total
-            }
-            boulder {
-              total
-            }
-            aid {
-              total
-            }
-          }        
-      }
-      metadata {
-        areaId
-        leaf
-        isBoulder
-        lat
-        lng 
-        leftRightIndex
-      }
-      pathTokens  
-      ancestors
-      climbs {
-        id
-        name
-        fa
-        grades {
-          font
-          french
-          vscale
-          yds
-        }
-        safety
-        type {
-          trad
-          tr
-          sport
-          mixed
-          bouldering
-          alpine
-          aid
-        }
-        metadata {
-          climbId
-          leftRightIndex
-        }
-        content {
-          description
-        }
-      }
-      children {
-        uuid
-        areaName
-        totalClimbs
-        metadata {
-          leaf
-          isBoulder
-        }
-        children {
-          uuid
-        }
-      }
-      content {
-        description 
-      }
-      updatedAt
-      updatedBy
-      createdAt
-      createdBy 
-    }
-  }`
 
-  const rs = await graphqlClient.query<{ area: AreaType }>({
-    query,
+  const rs = await graphqlClient.query<{ area: AreaType, getAreaHistory: ChangesetType[] }>({
+    query: QUERY_AREA_BY_ID,
```

Comment:
Sorry about the extraordinarily belated response.

I think it's a great idea to use `QUERY_AREA_BY_ID` instead of inlining the long GQL query in the page.  However, the crag page is crashing due to undefined query fields. `QUERY_AREA_BY_ID` hasn't been updated lately to match the inline query.  All you need to do is updating QUERY_AREA_BY_ID with the deleted inline query.

Are you still available to fix this?  If not please let me know.  I'll update your PR by the end of this coming week.

Thank you!

Reply:
Hey @vnugent, sorry that I didn't catch that! I can update this pr today.